airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry.py

```diff
@@ -137,46 +191,72 @@ def generate_and_persist_registry(
 
 
 @asset(
-    required_resource_keys={"slack", "registry_directory_manager", "latest_oss_registry_entries_file_blobs", "latest_metrics_gcs_blob"},
+    required_resource_keys={
+        "slack",
+        "registry_directory_manager",
+        "latest_oss_registry_entries_file_blobs",
+        "release_candidate_oss_registry_entries_file_blobs",
+        "release_candidate_metadata_file_blobs",
+        "latest_metrics_gcs_blob",
+    },
     group_name=GROUP_NAME,
 )
 @sentry.instrument_asset_op
-def persisted_oss_registry(context: OpExecutionContext, latest_connnector_metrics: dict) -> Output[ConnectorRegistryV0]:
+def persisted_oss_registry(
+    context: OpExecutionContext,
+    latest_connector_metrics: dict,
```

Comment:
What do we expect to be in `latest_connector_metrics`?

Reply:
It's some metrics fetched from the data warehouse which end up feeding fields related to usage metrics and sync success.